l10n_br_hr/views/hr_employee_view.xml

```diff
@@ -14,7 +14,6 @@
                 <field
                     name="naturalidade"
                     options="{'no_create': True, 'no_open': True}"
-                    required="1"
```

Comment:
Oq acha de mudar o nome deste campo para ingles já ? nao verifiquei se tem outros campos em portugues.

Reply:
Vou olhar

Reply:
@marcelsavegnago tinha o tipo do funcionário também, corrigido

Reply:
@DiegoParadeda https://github.com/odoo/odoo/blob/499056a82db26f7d9caa86314e666e2bd49cc79c/addons/hr/models/hr_employee.py#L70

Reply:
campo country_of_birth já nativo.. acho que podemos usar ele.

Reply:
e ainda tem o place_of_birth que é char

Reply:
talvez o place_of_birth pode ser usado no lugar do birth_city ou até mesmo ser populado com o nome da cidade caso tenha que ter um campo birth_city.

enfim, acho valido dar uma geral no modelo nativo e ver se tem mais alguma coisa que pode ser utilizado.

Reply:
Eu vi esses outros campos sim, só mantive por ser o único campo do modelo **res.city**.
Mas assim, o endereço do res partner já tem todos esses campos disponíveis então daria até pra remover o campo

Reply:
@DiegoParadeda não acho que o endereço serve como substituto.

Reply:
Acho melhor manter o campo neste caso, e se for manter o campo então adiciona o _id no nome do campo